docs/maya/muscle.md

```diff
@@ -212,14 +212,19 @@ To enable the debugger the *Debug* checkbox must be marked. To select the specif
   <figcaption><b>Figure 9</b>: AdnMuscle debug features. From left to right: Muscle Fibers, Attachment To Transform Constraints, Slide On Segment Constraints, Attachment To Geometry Constraints, Slide On Geometry Constraints and Shape Preservation.</figcaption>
 </figure>
 
+<figure markdown>
+  ![muscle fibers activation debug](images/muscle_fibers_activation_debug.png)
```

Comment:
There are some screenshots that still require updating to show the new Debug Fibers section. As we are also required to update the Initialization Settings screenshots it might be worth to document all of this in the Uber issue and update these screennshots at the end. For example: `setup_muscle_4`